playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { expect, Page, test } from '@playwright/test';
+import HomePage from 'tests/singular/home/home-page';
+import * as nav from 'tests/singular/lib/nav';
+import * as auth from 'tests/lib/auth-singular';
+import { fillEmailPassword } from 'tests/lib/auth-singular';
+import MedicalRecordReleaseForm from './medical-record-release-form';
+import PatientSurveyPage from './patient-survey-page';
+import PreScreeningPage from './pre-screening-page';
+import EnrollMyAdultDependentPage from './enroll-my-adult-dependent-page';
+import ConsentFormForAdultDependentPage from './consent-form-for-adult-dependent-page';
+import { makeEmailAlias } from '../../../utils/string-utils';
```

Comment:
`import` relative path is discouraged. This becames 
`import { makeEmailAlias } from 'utils/string-utils';`